examples/python/testcase2.py

```diff
@@ -9,22 +9,14 @@
 
 # GENERAL PACKAGE IMPORT
 # ----------------------
-import requests
-import time
-from examples.python.custom_kpi import custom_kpi_calculator as kpicalculation
-import json,collections
-import pandas as pd
+import sys
```

Comment:
Similar comment as before on ``testcase1.py`` doc string.  Check doc string description of this ``testcase2.py`` module, since it is the same as the ``interface.py`` module.

Reply:
I updated the doc string

Reply:
In testcase2 doc string, please change "feedback controller" to "supervisory controller".